apps/meteor/client/views/admin/info/OfflineLicenseModal.tsx

```diff
@@ -35,8 +41,8 @@ const OfflineLicenseModal = ({ onClose, license, licenseStatus, ...props }) => {
 	const handleApplyLicense = useMutableCallback(async () => {
 		setIsUpdating(true);
 		setLastSetLicense(newLicense);
-		const data = await addLicense({ license: newLicense });
-		if (data.success) {
+		const data = (await addLicense({ license: newLicense })) as unknown as { success: true };
```

Comment:
No need to check the response object because the `APIClient` implicitly consumes the `success` field and throws an error if `response.success !== true`. In any case you need to handle the request error, it's a better option to use `useEndpoint` hook directly.